lib/src/main/java/com/smileidentity/models/Bvn.kt

```diff
@@ -0,0 +1,110 @@
+package com.smileidentity.models
+
+import com.smileidentity.SmileID
+import com.smileidentity.networking.calculateSignature
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+private const val NIGERIA = "NG"
+private const val NIGERIA_BVN = "BVN_MFA"
+
+@JsonClass(generateAdapter = true)
+data class BvnToptRequest(
```

Comment:
nit: typo

Should be "OTP", not "OPT"
```suggestion
data class BvnTotpRequest(
```

Reply:
Fixed